darkwolf/theme.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "darwolf",
+    "version": "5.0",
+    "description": "Default phpMyAdmin theme",
```

Comment:
I hope we can come up with a slightly better description

Reply:
> I hope we can come up with a slightly better description

Hi @ibennetch. Thanks for your comments i create a new issue and upload the files with the fix json